jaxws-ri/httpspi-servlet/exclude.xml

```diff
@@ -0,0 +1,15 @@
+<!--
```

Comment:
It may be better to rename this file to spotbugs-exclude.xml to associate it with plugin.
Won't repeat this for other exclude files again.